src/ompl/tools/thunder/SPARSdb.h

```diff
@@ -887,7 +897,9 @@ namespace ompl
             /** \brief optional variable. If set, will sample each segment in the path with this many nodes and attempt to insert */
             std::optional<int> pathSamplingFactor_ {std::nullopt};
 
-            bool useCostInRoadmap_ {false};
+            bool useCostInRoadmap_{false};
+
+            bool addEdgeWithCost_{false};
```

Comment:
Can we add a comment to clarify the difference? I.e., as I assume:
- `useCostInRoadmap_: use the given edge weights in the graph when planning with A*`
- `addEdgeWithCost_: For a new node in the graph, compute the actual cost value for new edges, rather than assigning unit weight`

Reply:
You are right, this is super confusing. I added some docstrings